pkg/formatter/table.go

```diff
@@ -72,11 +72,13 @@ func WriteNodeReport(records []*reporting.NodeReportItem) {
 	lines := strings.SplitN(bufStr, "\n", 2)
 	width := tablewriter.DisplayWidth(lines[0])
 
+	var warning strings.Builder
 	fmt.Printf(bufStr)
 	if termWidth < width {
-		fmt.Print("\nNote:  To view the report with correct formatting, please expand")
-		fmt.Printf("\n       your terminal window to be at least %v characters wide\n", width)
+		warning.WriteString(fmt.Sprintf("\nNote:  To view the report with correct formatting, please expand"))
```

Comment:
`Sprintf()` is not needed here.
```suggestion
		warning.WriteString("\nNote:  To view the report with correct formatting, please expand")
```